kernel/solo5.h

```diff
@@ -136,8 +165,13 @@ uint64_t solo5_clock_monotonic(void);
 uint64_t solo5_clock_wall(void);
```

Comment:
would be great to specify error behaviour for these as well... at least the ukvm implementation asserts that `clock_gettime` returns non-zero